src/lib/assets/yaml.ts

```diff
@@ -40,7 +40,7 @@ export function zarfYaml({ name, image, config }: Assets, path: string) {
   return dumpYaml(zarfCfg, { noRefs: true });
 }
 
-export async function allYaml(assets: Assets) {
+export async function allYaml(assets: Assets, rbacMode: string = "") {
```

Comment:
Similar to question in src/lib/assets/index.ts, what's the purpose of a default empty string for rbacMode here?  Seems a non-default-valued param would be better here because it wouldn't imply that "" was an expected value.  :thinking: